### DISCUSSION TITLE
Thumbnail image attribute in production list

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#3241] [Questions & Answers]
with the "Configurable Swatches" function it is possible to highlight an image associated with a specific attribute in the product list of the categories and within the product page.

This is only possible if you make configurable products.

My question is as follows:

Is there a way or a component that can print an image associated with an option of an attribute in the product list?

a concrete example with an existing attribute the colors.

Suppose I sell a t-shirt that has three main colors printed on the fabric (white, yellow, red) and that I want to highlight them in the product list, but the product is not a configurable product but a simple product.

it's possible ? if so how?

**Comment by fballiano** [comment]
mmm I'd create a combobox attribute for the color (actually I think it exists by default) and then modify the templates in order to print the color as an image

**Reply by Danieleeffe1** [reply]
I understood that to display the color of the t-shirt (I'm using it as an example) you need to upload an image with the same name as the attribute to the media/wysiwyg/SWATCHES folder.

perhaps adding a string of code that calls a swatches image for the attributes visible on the product page?
